config/webpack/webpack.config.js

```diff
@@ -231,10 +232,11 @@ const buildWebpackConfigs = builds.map(
         name: 'render',
         mode: webpackMode,
         entry: {
-          render:
-            paths.renderEntry || path.join(__dirname, '../server/server.js')
+          render: paths.renderEntry
         },
         target: 'node',
+        // Don't bundle or transpile anything in node_modules
+        externals: [nodeExternals()],
```

Comment:
We'll probably need exceptions here for `seek-style-guide` and libraries defined in the `compilePackages` option.

Reply:
👍